include/sys/euiAnimator.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <nn/font/font_Util.h>
+
+#include <nn/ui2d/ui2d_Animation.h>
+#include <nn/ui2d/ui2d_Group.h>
+#include <nn/ui2d/ui2d_Pane.h>
+
+#include <eui/lytex/euiLayoutEx.h>
+
+namespace eui {
+
+class Animator : public AnimTransformBasic {

```

Comment:
Is AnimTransformBasic from ui2d? If so you're missing the namespace or class

Reply:
AnimTransformBasic is part of ui2d_Animation. You're right, I should probably fix that.